.editorconfig

```diff
@@ -18,7 +18,7 @@ indent_size = 4
 tab_width = 4
 
 # New line preferences
-end_of_line = lf
+end_of_line = crlf
```

Comment:
Keep 'LF' because all files are currently formatted with LF. You just need to run `dotnet format` and your code will be formatted according to the `.editorconfig`.